src/components/LoginForm.js

```diff
@@ -76,7 +77,13 @@ export default class LoginForm extends React.Component {
         if (err) {
           this.setState({ errorMessage: err.message });
         } else {
-          this.history.pushState(null, this.props.redirectTo || '/');
```

Comment:
Before, there was only one line in this `else` block, now there's more. I would recommend to return early in `if` and remove the `else` statement. That way you can keep using `var` instead of `let` too :)